common/templates/emails/new_enrollment_email_body.txt

```diff
@@ -0,0 +1,11 @@
+<%! from django.utils.translation import ugettext as _ %>${_("Enrollment received for:")}
+
+${_("Learner:")} ${user.username} \
+% if user_full_name:
+(${user_full_name})
+% else:
+
+% endif
+${_("Course:")} ${course_url}
+
+${_("More learner data in admin:")} ${user_admin_url}
```

Comment:
@clemente I hate to be fussy about this, but I think upstream will be too, so it's better to sort it out now.

When translating a line of text, you need to include all of the pieces so they can be translated all together.  Some languages may switch the order around completely, or change punctuation, or anything.

So for example, this file should be:
```
${_("Learner: {username} ({full_name})").format(
    username=user.username,
    full_name=user_full_name,
)}
${_("Course: {course_url}").format(course_url=course_url)}

${_("More learner data in admin: {user_admin_url}").format(user_admin_url=user_admin_url}
```

See e.g. https://github.com/edx/edx-platform/blob/6d2468d0dcf782a7561ecc61946146d6fae2762f/lms/templates/sysadmin_dashboard_gitlogs.html#L133
https://github.com/edx/edx-platform/blob/c0c740ad1d98b3b40fb3ce91620ec7faa7460b93/lms/templates/dashboard.html#L231